cpp/log/ct_extensions.h

```diff
@@ -22,6 +37,8 @@ extern const char kEmbeddedSCTListOID[];
 extern const char kPoisonOID[];
 // Extended Key Usage value for Precertificate signing
 extern const char kPrecertificateSigningOID[];
+// Name constrained intermediate CA may not be logged
+extern const char kNameConstraintNologIntermediateOID[];
```

Comment:
I suppose you're doing like the others here, but I'm wondering... is there a reason to have those visible in the header, rather than ensconced in the warmth of an anonymous namespace, in the `.cc` file?